public/app/pipeline-orchestrator.js

```
@@ -10,6 +10,83 @@
+    ? manifest.worklets.dsp_processor.sources
+    : [
+        { provider: 'vercel-blob', url: 'https://blob.vercel-storage.com/voiceisolate-pro/worklets/dsp-processor.js' },
+        { provider: 'r2',          url: 'https://models.voiceisolatepro.com/worklets/dsp-processor.js' },
+        { provider: 'huggingface', url: 'https://huggingface.co/Joker5514/voice-isolate-models/resolve/main/worklets/dsp-processor.js' },
+      ];
+
+  let lastErr;
+  for (const src of sources) {
+    if (src.provider === 'same-origin') continue; // already tried in step 1
+    try {
+      const resp = await fetch(src.url, { mode: 'cors', credentials: 'omit' });
```

ESLint

[error] 'fetch' is not defined.

(no-undef)